client/dive-common/components/Viewer.vue

```diff
@@ -43,18 +43,28 @@ import { cloneDeep } from 'lodash';
 import { getResponseError } from 'vue-media-annotator/utils';
 import context from 'dive-common/store/context';
 
+export interface ImageDataItem {
+  url: string;
+  filename: string;
+}
+
+function loadImageFunc(imageDataItem: ImageDataItem, img: HTMLImageElement) {
+  // eslint-disable-next-line no-param-reassign
+  img.src = imageDataItem.url;
+}
```

Comment:
This is identical to the default prop value. Why is it specified here?

Reply:
I believe it's a remnant of the refactor, I'll remove it.